content/600-about/200-prisma-docs/20-style-guide/01-writing-style.mdx

```diff
@@ -30,9 +30,9 @@ Write in a calm, assured tone of voice. Our tone is friendly and direct, but [we
 
 US English is the most globally recognized form of English. Use US English punctuation, grammar, and spelling. For example:
 
-- "color" over "colour"
-- "behavior" over "behaviour"
-- "Prisma plans to" over "Prisma plan to"
+- `color` over `colour`
```

Comment:
ℹ️ I put these behind back-ticks because otherwise `behaviour` would show as a spelling error. But I didn't want to add it to the dictionary because it's a specific case we want to catch. Putting the terms in back-ticks allows them to be there without a spelling issue being raised and without compromising readability IMO.